xml/System.Net.Sockets/UnixDomainSocketEndPoint.xml

```diff
@@ -167,9 +167,10 @@
         </Parameter>
       </Parameters>
       <Docs>
-        <param name="obj">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="obj">The <see cref="T:System.Object" /> to compare with the current <see cref="T:System.Net.Sockets.UnixDomainSocketEndPoint" />.</param>
+        <summary>Determines whether the specified <see cref="T:System.Object" /> is equal to the current <see cref="T:System.Net.Sockets.UnixDomainSocketEndPoint" />.</summary>
+        <returns>
+          <see langword="true" /> if the specified <see cref="T:System.Object" /> is equal to the current <see cref="T:System.Net.Sockets.UnixDomainSocketEndPoint" />; otherwise, <see langword="false" />.</returns>
```

Comment:
```suggestion
          <see langword="true" /> if the specified object is equal to the current <see cref="T:System.Net.Sockets.UnixDomainSocketEndPoint" />; otherwise, <see langword="false" />.</returns>
```